homeassistant/components/overkiz/coordinator.py

```diff
@@ -193,8 +210,10 @@ async def on_execution_registered(
     if event.exec_id and event.exec_id not in coordinator.executions:
         coordinator.executions[event.exec_id] = {}
 
+    # Set update interval to one second to receive faster
+    # state updates after command execution for stateful devices
     if not coordinator.is_stateless:
-        coordinator.update_interval = timedelta(seconds=1)
+        coordinator.set_update_interval(timedelta(seconds=1))
```

Comment:
We don't allow 1 second update interval. Min 5 seconds for polling integrations.

Reply:
This is current behavior in the integration (thus no new behavior), and this only happens for a maximum of 10 seconds. This is done to get instant state updates after executing a command on a stateful device. During this period it will poll every second to have instant state updates (and than restore to original interval). Somfy agreed with this by the way, so we are not overloading their service. (they actually do the same in their application...).

Would this be allowed in Home Assistant? There is no possibility to use a websocket or other subscriber/push method unfortunately.

Reply:
No, we don't allow that. The service handler should just request a refresh. We shouldn't change the interval.